### PR TITLE
Appveyor.yml

### PR DESCRIPTION
**Pull request by schneems**
cc/ @maclover7

### CODE REVIEW

**Comment by schneems** [comment]
Doesn't look like this PR triggered a build at https://ci.appveyor.com/project/schneems/sprockets-8q1hv

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+install:
+  - set PATH=C:\Ruby22\bin;%PATH%
+  - bundle install
+
+build: off
```
Comment (kaspth) [inline appveyor.yml]: Could this be why it didn't build? 😁

Comment (schneems) [inline appveyor.yml]: Removed that line.

**Comment by rafaelfranca** [comment]
Should be working now. Let me try to close and reopen.

**Comment by rafaelfranca** [comment]
Yeah, it is running.

**Comment by schneems** [comment]
🚀

**Comment by schneems** [comment]
```
Specify a project or solution file. The directory does not contain a project or solution file.
```

**Comment by maclover7** [comment]
🚀 Love this!!

Just for reference, here's Rubygems' AppVeyor configuration file: https://github.com/rubygems/rubygems/blob/master/appveyor.yml

**Comment by maclover7** [comment]
Just for another example, here's npm's configuration file: https://github.com/npm/npm/blob/iarna/windows-wip/appveyor.yml

**Comment by schneems** [comment]
Posted on their forums http://help.appveyor.com/discussions/problems/4585-specify-a-project-or-solution-file-the-directory-does-not-contain-a-project-or-solution-file

**Comment by FeodorFitsner** [comment]
To disable build step add this to `appveyor.yml`:

    build: off

**Comment by schneems** [comment]
Disable the build as in turn off testing? I had that line previously but tests weren't running.

**Comment by FeodorFitsner** [comment]
Complete appveyor.yml will look like:

```
install:
  - set PATH=C:\Ruby22\bin;%PATH%
  - bundle install

build: off

before_test:
  - ruby -v
  - gem -v
  - bundle -v

test_script:
  - bundle exec rake

environment:
  matrix:
    - RUBY_VERSION: 23
    - RUBY_VERSION: 22
    - RUBY_VERSION: 21
    - RUBY_VERSION: 200
```

**Comment by schneems** [comment]
All tests are green, shipping this, thanks for the help 🚀

**Comment by maclover7** [comment]
🙌 nice work!! :)

**Comment by Sherlock1x** [comment]
Keep up the great work happy coding..........
040716niburu

**Pull request merged**